### DISCUSSION TITLE
How to make wildcard aliases

### DISCUSSION THREAD

**Discussion by mildred** [stalwartlabs/jmap-server#26] [Q&A] [answered]
Just a simple question because the docs does not talk about it. is there a way to make a catch-all alias to collect every mail for a domain?

Thank you

**Comment by mildred** [comment, marked as answer]
Seems to be implemented here in `expand_rcpt` :

https://github.com/stalwartlabs/jmap-server/blob/87cc39c3cf21daabc93c170574f5839c0cfdc539/components/jmap_sharing/src/principal/account.rs

I see no trace of wildcards, the database query is done once with the full e-mail address with an equality operator.

**Reply by mdecimus** [reply]
This feature is now being tracked at https://github.com/stalwartlabs/jmap-server/issues/27.
